lib/src/source_visitor.dart

```diff
@@ -68,20 +68,19 @@ class SourceVisitor extends ThrowingAstVisitor {
   /// in unison.
   final List<MetadataRule> _metadataRules = [];
 
-  /// The mapping for collection literals that are managed by the argument
-  /// list that contains them.
+  /// The mapping for blocks that are managed by the argument list that contains
+  /// them.
   ///
-  /// When a collection literal appears inside an [ArgumentSublist], the
-  /// argument list provides a rule for the body to split to ensure that all
-  /// collections split in unison. It also tracks the chunk before the
-  /// argument that determines whether or not the collection body is indented
-  /// like an expression or a statement.
+  /// When a block expression, such as a collection literal or a multiline
+  /// string, appears inside an [ArgumentSublist], the argument list provides a
+  /// rule for the body to split to ensure that all blocks split in unison. It
+  /// also tracks the chunk before the argument that determines whether or not
+  /// the block body is indented like an expression or a statement.
   ///
-  /// Before a collection literal argument is visited, [ArgumentSublist] binds
-  /// itself to the left bracket token of each collection literal it controls.
-  /// When we later visit that literal, we use the token to find that
-  /// association.
-  final Map<Token, ArgumentSublist> _collectionArgumentLists = {};
+  /// Before a block argument is visited, [ArgumentSublist] binds itself to the
+  /// left bracket token of each collection literal it controls. When we later
```

Comment:
"left bracket token of each collection literal it controls" → "beginning token of each block it controls".